SustainabilityOnMars/GuidedNotebooks/pets-in-space/challenge-0.ipynb

```diff
@@ -0,0 +1,137 @@
+      "text/html": [
+       "\n",
+       "        <iframe\n",
+       "            width=\"400\"\n",
+       "            height=\"300\"\n",
+       "            src=\"https://www.youtube.com/embed/P6KNkb7xVjk\"\n",
+       "            frameborder=\"0\"\n",
+       "            allowfullscreen\n",
+       "        ></iframe>\n",
+       "        "
+      ],
+      "text/plain": [
+       "<IPython.lib.display.YouTubeVideo at 0x7f81ddcc7190>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "YouTubeVideo('P6KNkb7xVjk') "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#❗️Run this cell\n",
+    "i.userinfo()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#❗️Run this cell\n",
+    "i.challengeplotall()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 📚 Challenge Complete!\n",
+    "\n",
+    "Wow, great job! Now that you've gotten started, continue to Challenge 1 in the next notebook!"
```

Comment:
You can hyperlink to the next notebook so "... continue to [Challenge 1 in the next notebook](./challenge-1.ipynb)!"